src/features/rewards/walletSummarySlider/style.ts

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+
+export const StyledWrapper = styled<{}, 'div'>('div')`
+  display: block;
+  width: 100%;
+` as any
+
+export const StyledTransitionWrapper = styled<{}, 'div'>('div')`
```

Comment:
Please send props in and change `(p: {show: boolean})` into `p`

example https://github.com/brave/brave-ui/blob/2f6ca216aeac98ab27f8ceb2004c6b28f6565ec4/src/features/rewards/box/style.ts#L10